setup.cfg

```diff
@@ -117,6 +117,8 @@ style =
     # pyupgrade 2.8+ required for --py39-plus flag
     pyupgrade>=2.8,<4
 tests =
+    # hydra-core 1.1.0+ required for recursive instantiation
```

Comment:
What's the minimum version that works without recursive instantiation? Users can install a newer version if they want to, but if older versions work for most cases then I think we should set our minimum to that.